css/style.css

```diff
@@ -0,0 +1,272 @@
+}
+
+/* navbar */
+
+.navbar{
+    width: 100%;
+    height: auto;
+    padding: 31px 25px 43px 25px;
+}
+
+.navbar-container{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navbar-left{
+    flex-grow: 1;
+}
+
+.navbar ul{
+    display: flex;
+}
+
+.navbar-item{
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 25px;
+    font-family: 'Libre Baskerville', serif;
+    padding-left: 31px;
+}
+
+.navbar-input{
+    background: url(../images/search.svg);
+    background-repeat: no-repeat;
+    background-position: right 18px center;
+    width: 303px;
+    height: 46px;
+    border: 2px solid #1C1C1C;
+    border-radius: 10px;
+    padding: 12px 0;
+    padding-right: 18px;
+    padding-left: 18px;
+    border: 0px;
+    padding-right: 60px;
+}
+
+.navbar-btn{
+    width: 135px;
+    border: 2px solid #1C1C1C;
+    border-radius: 10px;
+    background-color: #ffffff;
+    font-family: 'Libre Baskerville', serif;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 25px;
+    text-align: center;
+    height: 46px;
+}
+
+.navbar-right > .navbar-item {
+    display: flex;
+    align-items: center;
+}
+
+/* media nav */
+
+.header-btn_media {
+    height: 40px;
+    /* border: 2px solid #000000; */
+    cursor: pointer;
+    display: none;
+    flex-direction: column;
+}
+
+.header-btn_media div {
+    width: 26px;
+    height: 2px;
+    background-color: #000000;
+    margin: 7px; 
+}
+
+.mobile-nav{
+    position: fixed;
+    top: 127px;
+    width: 90%;
+    height: 100%;
+    background-color: #ffffff;
+    z-index: 100;
+    right: 0;
+    transition: all 0.3s;
+
+    right: -100%;
+}
+
+.mobile-nav-items{
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 90px;
+}
+
+.mobile-nav-items li{
+    padding-bottom: 15px;
+}
+
+.mobile-nav-input{
+    background-position: left  center;
+    padding: 0 15px 0 35px;
+    width: 150px;
+}
+
+.hidden{
+    right: 0;
+}
+
+.no-scroll{
+    overflow: hidden;
+}
+
+/* post */
+
+.post{
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    height: 592px;
+    position: relative;
+}
+
+
+.post-content{
+    background-color: #ffffff;
+    max-width: 509px;
+    padding: 76px 40px 96px 39px;
+}
+
+.post-left{
+    left: 45px;
+    position: absolute;
+}
+
+.post-tag{
+    font-size: 20px;
+    line-height: 27px;
+    text-transform: uppercase;
+}
+
+.post-title{
+    font-family: 'Libre Baskerville', serif;
+    font-weight: 700;
+    font-size: 33px;
+    line-height: 41px;
+    margin-bottom: 12px;
+}
+
+.post-meta-wrapper{
+    display: flex;
+    align-items: center;
+}
+
+.post-meta{
+    line-height: 22px;
+    color: rgba(28, 28, 28, 0.5);    
+}
+
+.post-meta__circle{
+    width: 3px;
+    height: 3px;
+    border-radius: 1.5px;
+    display: inline-block;
+    background: rgba(28, 28, 28, 0.5);
+    margin: 0 10px;
+}

```

Comment:
previously you had:
```
.post-meta__content::before{
    position: absolute;
    content: ".";
    bottom: 5px;
    left: -13px;
}
```

my suggestion to use same way, but make a circle with this pseudo element instead of using dot character or another html element.
Like this it's main rules, se here need to add alignment styles
```
.post-meta__content::before{
    content: "";
    width: 3px;
    height: 3px;
    border-radius: 1.5px;
    background-color: #000;
}
```

Reply:
I deleted separate resource files and use pseudo elements for circle elements and vertical elements like you described